root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4163,6 +4163,24 @@ limited_link_type_combinations: [
              input_entity_type: 'artist',
     expected_relationship_type: 'otherdatabases',
   },
+  {
+                     input_url: 'https://openlibrary.org/publishers/ISIS_Audio_Books',
+             input_entity_type: 'label',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://openlibrary.org/publishers/ISIS_Audio_Books',
+  },
```

Comment:
This test is pretty redundant with the next one which also has underscores (`_`) in the slug name. I would suggest to drop this one which has no comma (`,`) in it. Note that `expected_clean_url` is tested to be stable through the function `clean` so the scheme `https` is tested too by the next test.